addons/mail/static/src/core/common/call_notification_message.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.CallNotificationMessage">
+        <div class="d-flex justify-content-center align-items-end gap-1">
+            <i class="fa fa-lg fa-phone pt-1 text-success me-1"/>
+            <t t-if="props.author === store.self">You</t>
+            <span t-else="" class="o-mail-Message-author cursor-pointer o-hover-text-underline" t-on-click="ev => this.onClickAuthor(ev)">
+                <strong class="cursor-pointer" t-out="props.author.name"/>
+            </span>
+            <span t-esc="callInformation" />
+            <span class="text-muted opacity-75 smaller" t-esc="props.messageDate" />
+        </div>
+    </t>
```

Comment:
`onClickAuthor` is not defined. I would simply remove this feature: this makes the notification too complex: a notification should be as brief as possible. The name is useful when someone initiated the call, no need to make it clickable because the person making the call is immediately available on given channel.

And at end of call, there's no need to name the caller of the call that was made. This makes the notification too verbose and unbalances the message list. I wouldn't mind the notification being clickable and you see this information if you want to know it, but I don't like it being necessarily present and makes it too verbose.